client/landing/stepper/declarative-flow/internals/steps-repository/domains/style.scss

```diff
@@ -471,10 +471,18 @@
 		}
 	}
 
+	.register-domain-step__search-card {
+		background: none;
+	}
+
 	.search-component {
 		overflow: hidden;
 	}
 
+	.search-filters__dropdown-filters {
+		display: none;
+	}
+
```

Comment:
Hi @fditrapani! I've checked with the team and we think that instead of hiding the filter with CSS, it would be better to not render it at all. That can be done by adding this conditional to the beginning of the `renderSearchFilters` method [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/domains/register-domain-step/index.jsx#L536):

```js
if ( [ HUNDRED_YEAR_PLAN_FLOW, HUNDRED_YEAR_DOMAIN_FLOW ].includes( this.props.flowName ) ) {
	return null;
}
```

What do you think?

Reply:
Love it! It's a much better approach. Thanks for the suggestion @leonardost!

I just spoke with Ramy, he said you can go ahead and update it when you have a moment.